cognite/client/data_classes/transformations/__init__.py

```diff
@@ -215,21 +215,21 @@ def try_get_or_create_nonce(oidc_credentials: OidcCredentials | None, project: s
                 else:
                     credentials = None
                 try:
-                    session = self._cognite_client.iam.sessions.create(credentials, project=project)
+                    session = self._cognite_client.iam.sessions.create(credentials)
                     ret = NonceCredentials(session.id, session.nonce, project)
                     sessions_cache[key] = ret
                 except Exception:
                     ret = None
             return ret
 
         if self.source_nonce is None and self.source_oidc_credentials:
-            project = self.source_oidc_credentials.cdf_project_name or self._cognite_client.project
+            project = self.source_oidc_credentials.cdf_project_name or self._cognite_client.config.project
```

Comment:
nice catch

Reply:
`mypy` found this one 😅  Credit where credit is due, etc 😜